src/examples/src/config.tsx

```diff
@@ -316,8 +320,18 @@ export const config = {
 		{ label: 'default', theme: { theme: {} } }
 	],
 	tests,
-	readmePath: (widget: string) => `src/${widget}/README.md`,
-	widgetPath: (widget: string, filename: string) => `src/${widget}/${filename || 'index'}.tsx`,
+	readmePath: (widget: string) => {
+		if (widget === 'vstack' || widget === 'hstack') {
```

Comment:
is this something we can raise as an issue against parade to allow for different readme paths per item

Reply:
Don't need that anymore, removed.